dev/install/install_dependencies_with_workarounds.py

```diff
@@ -1,35 +1,11 @@
 #!/usr/bin/env python
 # coding: utf-8
-# Check if the platform requires conda tensorflow.
+# Check if the platform requires conda skimage
 #
 # Author: Christian S. Perone
 
-import platform, subprocess
+import subprocess
 
-def tf_workarounds():
```

Comment:
not sure if we should remove

Reply:
Ah, so. The `dev/` folder is largely... dead code? There are a lot of things in that folder that were once relevant, but are no longer touched. SCT hasn't exactly had a great procedure for pruning bits that are no longer needed?

So, as far as changes that affect present-day SCT, you can safely ignore almost everything in `/dev/`